src/commentCount.test.js

```diff
@@ -0,0 +1,58 @@
+import { commentCounter } from './commentsRequest.js';
+
+const unmockedFetch = global.fetch;
+
+beforeAll(() => {
+  global.fetch = () => Promise.resolve({
+    json: () => Promise.resolve([
+      {
+        comment: 'I did it like three times',
+        creation_date: '2021-12-24',
+        username: 'Kevin',
+      },
+    ]),
+  });
+});
+
+afterAll(() => {
+  global.fetch = unmockedFetch;
+});
+
+test('It counts the right number of comments1', async () => {
+  const commentNumber = await commentCounter(52772);
+  expect(commentNumber).toEqual(1);
+});
+
+test('It counts the right number of comments1', async () => {
+  global.fetch = () => Promise.resolve({
+    json: () => Promise.resolve([
+      {
+        comment: 'I did it like three times',
+        creation_date: '2021-12-24',
+        username: 'Kevin',
+      },
+      {
+        comment: 'I did it like three times',
+        creation_date: '2021-12-24',
+        username: 'Kevin',
+      },
+      {
+        comment: 'I did it like three times',
+        creation_date: '2021-12-24',
+        username: 'Kevin',
+      },
+    ]),
+  });
+
+  const commentNumber = await commentCounter(52772);
+  expect(commentNumber).toEqual(3);
+});
+
+test('It counts the right number of comments2', async () => {
+  global.fetch = () => Promise.resolve({
+    json: () => Promise.resolve(Error),
+  });
+
+  const commentNumber = await commentCounter(52772);
+  expect(commentNumber).toBe(0);
+});
```

Comment:
The test fails when running `npm run test`

![Screenshot 2021-12-24 at 09 17 27](https://user-images.githubusercontent.com/31847346/147333676-5c0d3faa-c57e-4beb-855e-cbf87e5f0325.png)